appveyor.yml

```diff
@@ -6,7 +6,7 @@ environment:
 
   global:
     PYTHON: "C:\\conda"
-    CONDA_DEPENDENCIES: "numpy seaborn matplotlib sphinx pillow pytest pytest-cov sphinx_rtd_theme joblib"
+    CONDA_DEPENDENCIES: "numpy seaborn matplotlib sphinx pillow pytest pytest-cov sphinx_rtd_theme joblib plotly pandas"
```

Comment:
I don't think these are needed as pandas was already being used in an example and it wasn't installed. Though I see we build with no-plot but also do 'make html' ? Maybe @larsoner can clarify?

Reply:
I think it is still needed for the appveyor build. However I can remove pandas since it is a dependency of seaborn, hence the example with pandas was working before (thanks to seaborn)